docs/how/auth/add-users.md

```diff
@@ -47,7 +52,8 @@ For example, to mount a user.props file that is stored on my local filesystem at
     .....
     # The new stuff
     volumes:
-      - <path-to-your-user.props>:/datahub-frontend/conf/user.props
+      - ${HOME}/.datahub/plugins:/etc/datahub/plugins
```

Comment:
have we tested this? where we have /etc/datahub/plugins being mounted and then another directory inside of this directory /etc/datahub/plugins/auth also being mounted

Reply:
No - I've not tested this yet. I should though, thanks! The issue with removing the original mounting is that it could have other stuff in it also later down the road

Reply:
I did test this. It works!